src/core/xfa/template.js

```diff
@@ -2462,14 +2462,17 @@ class Fill extends XFAObject {
 
   [$toStyle]() {
     const parent = this[$getParent]();
+    const style = Object.create(null);
+
     let propName = "color";
     if (parent instanceof Border) {
       propName = "background";
     }
-    if (parent instanceof Rectangle) {
+    if (parent instanceof Rectangle || parent instanceof Arc) {
```

Comment:
Is Arc needed for the same PDF? Or is it a defensive change?

Reply:
yep it is.